app/services/user_update.rb

```diff
@@ -24,6 +24,49 @@ def update
 
 private
 
+  class SupportedPermissionsSanitizer
```

Comment:
Wouldn't this be better housed in the `lib` folder? Similar to:

https://github.com/alphagov/signon/blob/master/lib/user_parameter_sanitiser.rb
and
https://github.com/alphagov/signon/blob/master/lib/unsupported_permission_error.rb